source/index.html.md

```diff
@@ -496,185 +496,6 @@ To perform this operation, you must be authenticated by means of one of the foll
 bearerAuth
 </aside>
 
-## Onboard Completed
```

Comment:
@ron-bierman-unity why did this get removed?

Reply:
@justinhamadeunity I removed it in the swagger doc.  We don't use it anymore.